src/pages/home/Home.js

```diff
@@ -0,0 +1,69 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import PostTile from "../../components/postTile/PostTile";
+import RedditHeader from "../../components/redditHeader/RedditHeader";
+import './Home.css';
+import redditLogo from "../../assets/redditlogo.png";
+import NoviFooter from "../../components/noviFooter/NoviFooter";
+
+function Home({error, toggleError, loading, toggleLoading}) {
+    const [redditPosts, setRedditPosts] = useState({});
```

Comment:
Zodra de setter is aangeroepen is `redditPosts` niet meer van het type object, maar array. Ik zou daarom de default value aanpassen naar een empty array:
```suggestion
    const [redditPosts, setRedditPosts] = useState([]);
```